R/reactiveGeneSetDb-module.R

```diff
@@ -57,7 +57,7 @@ reactiveGeneSetDb <- function(input, output, session, gdb,
   })
 
   shiny::observe({
-    gs.range <- input$size
+    gs.range <- shiny::req(input$size)
```

Comment:
This one broke the shiny app - fun to track down from all the way up that chain.

Reply:
Thank you for chasing this down!